frontend/src/components/CourseDescriptionPanel/CourseAttributes/CourseAttributes.tsx

```diff
@@ -109,6 +116,62 @@ const CourseAttributes = ({ course, courseCapacity }: CourseAttributesProps) =>
         {
           title: 'Units of Credit',
           content: UOC
+        },
+        {
+          title: 'Unilectives Rating',
+          content: rating ? (
+            <>
+              <S.RatingWrapper>
+                <S.DialWrapper>
```

Comment:
I think these are good for now, but in the future if I were to refactor this, I'd consider moving the dials to a cute lil subcomponent, as they are all the same formatting with different values.

Reply:
but for now, is ok to resolve this

Reply:
Alright, can change later then.